src/MeetingsSDKAdapter.test.js

```diff
@@ -717,7 +723,6 @@ describe('Meetings SDK Adapter', () => {
       await meetingSDKAdapter.handleLocalShare(meetingID);
 
       expect(mockSDKMeeting.updateShare).toHaveBeenCalled();
-      expect(meetingSDKAdapter.meetings[meetingID].localShare).toBeNull();
```

Comment:
in order to leave this `expect` in place, we need to wait a little more. You can use https://www.npmjs.com/package/wait-for-expect

Reply:
I don't think this should be here anymore since here we call the handleLocalShare method and we check if the disableSharingStream is triggered and this doesn't set the localShare property to null anymore.